core/market/decentralized/tests/utils/mock_offer.rs

```diff
@@ -16,9 +26,56 @@ pub fn example_offer() -> Offer {
 pub fn example_demand() -> Demand {
     let properties = serde_json::json!({
         "golem": {
+            "golem.node.debug.subnet": "blaa".to_string(),
             "node.id.name": "itstest".to_string(),
             "srv.comp.wasm.task_package": "test-package".to_string(),
         },
     });
     Demand::new(properties, "(golem.node.debug.subnet=blaa)".to_string())
 }
+
+impl MarketNode {
```

Comment:
is it intentionally here and not is `mock_node.rs`?

Reply:
Yes, these functions are loosely related to test framework. They are rather functions with rank of example_offer or demand.